packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -624,12 +637,32 @@ export class ConnectionFlow {
     }
   }
 
+  watchTriggerJobs() {
+    // can happen when the current trigger comes from realtime
```

Comment:
to not have `this.trigger` set? 

I don't understand clearly this comment

Reply:
The comment finished in the wrong place, sorry. It is `this.trigger?.current_state` which can be empty